.travis.yml

```diff
@@ -1,35 +1,62 @@
 language: php
+sudo: false
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+        - $HOME/symfony-bridge/.phpunit
 
-php:
-    - 5.5
-    - 5.6
-    - 7.0
-    - 7.1
-    - 7.2
-    - nightly
+env:
+    global:
+        - PHPUNIT_FLAGS="-v"
+        - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
 
 matrix:
+    fast_finish: true
     include:
+          # Minimum supported Symfony version with the latest PHP version
+        - php: 7.2
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
+
+          # Test the latest stable release
+        - php: 5.5
         - php: 5.6
-          env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
+        - php: 7.0
         - php: 7.1
-          env: DEPENDENCIES=dev SYMFONY_DEPRECATIONS_HELPER=weak
-    allow_failures:
-        - php: nightly
-
-sudo: false
+        - php: 7.2
+          env: COVERAGE=true PHPUNIT_FLAGS="-v --coverage-clover=coverage.clover"
+
+          # Test LTS versions. This makes sure we do not use Symfony packages with version greater
```

Comment:
this should be `Test each symfony major version`. These jobs don't cover only LTS (there is no v4 LTS yet, it will come in 2 years)